frontend/packages/shared/src/types/AppConfig.ts

```diff
@@ -2,7 +2,7 @@ import type { SupportedLanguage, ValidLanguage } from './SupportedLanguages';
 
 export type AppConfig = {
   repositoryName: string;
-  serviceName: string;
+  serviceName: { nb: string; en: string; nn: string };
```

Comment:
We can use the same type here as for `serviceName` in `AppConfigNew`: 
```suggestion
  serviceName: SupportedLanguage;
```